ros2_generate_interface_docs/ros2_generate_interface_docs/templates/msg-index.html.em

```diff
@@ -0,0 +1,18 @@
+@{
+import time
+}@
+<html>
+  <head>
+    <title>@(package) Message / Service / Action Documentation</title>
+    <link type="text/css" rel="stylesheet" href="../msg-styles.css" />
+  </head>
+  <body>
+    <div id="container">
+      <h1>@(package) Message / Service / Action Documentation</h1>
+        @[for links, msg in zip(links, interface_list)]@
```

Comment:
Nitpick: `links` seems to be a weird name for this variable. I assume it contains a relative path?

Reply:
Renamed to `relative_path` f69a57edce4c2c8823983e88ddd7e5ac4cc220ef